src/audio/buffer.c

```diff
@@ -5,21 +5,19 @@
 // Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
 //         Keyon Jie <yang.jie@linux.intel.com>
 
-#include <stdint.h>
-#include <stddef.h>
-#include <errno.h>
-#include <sof/sof.h>
-#include <sof/spinlock.h>
-#include <sof/list.h>
-#include <sof/stream.h>
+#include <ipc/topology.h>
```

Comment:
Sorry for being annoying, but I want ot knokw what's the pattern here? Usually it's something like from most local to global includes. While it's obvious for stdlib headers, I see the `ipc` headers come before `sof`, imho sof are more internal, since all other folders in `include` dir are exported/public. Maybe order should be local, include/sof folder (project inernal), then others from include/ folder (alphabetically), then stdlib etc headers.

Reply:
Yes, your approach is better.
Changes applied.

Reply:
@tlauda Thanks